data/contribution-guidelines.md

```diff
@@ -0,0 +1,21 @@
+# i18nify-data Contribution Guidelines
+
+First off, thank you for contributing to i18nify-data! 🎉
+
+Read this before adding or modifying anything in this data. These guidelines exist to help create and maintain packages better. Feel free to propose changes to this document.
+
+## Before Contributing
+
+Before contributing a new data or the modifications to the existing data, make sure the data source is either standard or the most reliable data source for the specific data.
```

Comment:
How can one ensure the data they are adding are sourced from standard data source? can you state some examples here for better understanding?